Gaijinizer patch/Map196.txt

```diff
@@ -175,24 +184,27 @@ magic of tranformed species. It was not just
 
 \\C[10]今までの道のりで『同じ魔力を得ていた者』からな……
 > CONTEXT: Map196/events/1/pages/0/163/Dialogue/enemy17
-Hohoho...The place, \\N[1]. I borrowed the
-magic power. \\C[10]From "the person who has obtained
-the same magic" in the way up to now ...#MTLed
+Hohoho... In short, \\N[1] was borrowing his
+magic power \\C[10]from on the journey so far
+"same magic power obtained person"...#IHaveNoFuckingIdea
 > END STRING
 
 > BEGIN STRING
 \\N[1]………
 
 夜が明けるわ…………
 > CONTEXT: Map196/events/1/pages/0/174/Dialogue/other1
-\\N[1] ......... I can see the night .........#MTLed
+\\N[1]...
+
+The sun's coming up...#Ron
 > END STRING
 
 > BEGIN STRING
 こうして、僕の
 
 『目に見えない介入者』との脱出劇は、幕を閉じた……
 > CONTEXT: Map196/events/1/pages/0/211/Dialogue
-Thus, my. The escape play with "The Invisible Man" has
-closed the curtain ...#MTLed
+And so, the curtain closed on this chapter of my
+escape and the help I received from the unseen
+intervener...#RonIffyWording
```

Comment:
Rewording for flow:
>_**And so the curtain closed on this chapter of my
escape, with the help I received from my unseen
'friend'...**_